packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -36,4 +43,8 @@ export const styles = css`
         height: ${controlSlimHeight};
         align-self: center;
     }
+
+    .action-menu.focused {
+        display: block;
```

Comment:
Can you explain why the `display` needs to change when the action menu is focused?

Reply:
The `focused` class is added on the action menu button immediately before we call `focus()` on it. From what I can tell, `focus()` has no effect if the menu button is not visible when it's called, and we hide them by default.

As you Tab through potentially multiple menu buttons on the row, only the menu button that's focused (or associated with the focused cell) is shown:
![MenuButtonFocus](https://github.com/ni/nimble/assets/20709258/905d64c6-7b8b-4ecc-a43f-06c77cd3d98d)